src/applications/burial-poc-v6/constants.js

```diff
@@ -0,0 +1,18 @@
+export const RadioLabels = {
+  spouse: 'Spouse',
+  child: 'Child',
+  parent: 'Parent',
+  executorAdministratorEstate: 'Executor/Administrator of estate',
+  other: 'Other',
+  vaMedicalCenter: 'VA medical center',
+  stateVeteransHome: 'State Veterans home',
+  nursingHome: 'Nursing home under VA contract',
+  service: 'Service-connected death',
+  nonService: 'Non-service-connected death',
+  vaMC: 'VA medical center death',
+};
+
+export const ExpandingGroupClass =
+  'vads-u-border-color--primary-alt-light vads-u-border-left--4px vads-u-padding-left--2 vads-u-padding-y--0p5 vads-u-margin-left--neg2p5';
```

Comment:
I'm not sure what this looks like in the app, but I'm wondering if there is a design system component that should be used instead

Reply:
Am going to try and refactor to use [ExpandingGroup web-component](https://design.va.gov/storybook/?path=/docs/components-expandinggroup--default)

Reply:
It looks like you were able to use the ExpandingGroup component, should this be deleted now?